static/js/blame.js

```diff
@@ -190,6 +212,12 @@ var BlameStripHoverHandler = new (class BlameStripHoverHandler {
         BlamePopup.blameElement = null;
       }, 100);
     } else {
+      // We run this code on either "mouseenter", or on a "click" event where
+      // the click landed inside the blame strip. In the latter case we set
+      // keepVisible to pin the blame popup open until another click dismisses
+      // it, or a scroll event happens (since the popup doesn't move properly with
+      // scrolling).
+      this.keepVisible = (event.type == 'click');
```

Comment:
nit: I'd remove these parens but your call :)

Reply:
I like the parens so I kept them, but updated for all the rest of your comments. :)